contrib/images/Makefile

```diff
@@ -1,20 +1,31 @@
 RELAYER_TAG := $(shell grep '^ENV RELAYER_TAG' cosmos-relayer/Dockerfile | cut -f3 -d\ )
+BABYLON_FULL_PATH := $(shell git rev-parse --show-toplevel)
 
 all: babylond cosmos-relayer
 
 babylond: babylond-rmi
-	docker build --tag babylonlabs-io/babylond -f babylond/Dockerfile \
-    		$(shell git rev-parse --show-toplevel)
+	docker build --tag babylonlabs-io/babylond -f babylond/Dockerfile ${BABYLON_FULL_PATH}
+
+babylond-e2e:
+	docker build --tag babylonlabs-io/babylond -f babylond/Dockerfile ${BABYLON_FULL_PATH} \
+		--build-arg BUILD_TAGS="e2e"
 
 babylond-rmi:
 	docker rmi babylonlabs-io/babylond 2>/dev/null; true
 
+e2e-init-chain-rmi:
+	docker rmi babylonlabs-io/babylond-e2e-init-chain --force 2>/dev/null; true
+
+e2e-init-chain:
+	docker build -t babylonlabs-io/babylond-e2e-init-chain --build-arg E2E_SCRIPT_NAME=chain --platform=linux/x86_64 \
```

Comment:
I wonder whether ` --platform` shouldn't be configurable by some env variable, as when I run test locally on arm mac those fail due to native binding to bls libraries and changing platform fixes that

Reply:
I did not know that, I will add --platform tag to docker builds

Reply:
Can you check if this https://github.com/babylonlabs-io/babylon/pull/46/commits/997de7cfcee2063e3b92b011da73c8d467da8b3a
solved?

Reply:
hmm I have checked it locally and got:
```
> make build-docker
/Library/Developer/CommandLineTools/usr/bin/make -C contrib/images babylond
docker rmi babylonlabs-io/babylond 2>/dev/null; true
Untagged: babylonlabs-io/babylond:latest
Deleted: sha256:7cd3d4a92eff50f5490ea0636532b3fbbcea3f5585d98fb886b1e96423614339
docker build --tag babylonlabs-io/babylond -f babylond/Dockerfile /Users/konradstaniec/Work/babylon/babylon --platform=local
[+] Building 2.0s (3/3) FINISHED                                                                                                                                    docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                                                                                                0.0s
 => => transferring dockerfile: 2.22kB                                                                                                                                              0.0s
 => ERROR [internal] load metadata for docker.io/library/debian:bookworm-slim                                                                                                       2.0s
 => ERROR [internal] load metadata for docker.io/library/golang:1.21                                                                                                                2.0s
------
 > [internal] load metadata for docker.io/library/debian:bookworm-slim:
------
------
 > [internal] load metadata for docker.io/library/golang:1.21:
------
Dockerfile:1
--------------------
   1 | >>> FROM golang:1.21 AS build-env
   2 |
   3 |     # Customize to your build env
--------------------
ERROR: failed to solve: golang:1.21: no match for platform in manifest: not found
make[1]: *** [babylond] Error 1
make: *** [build-docker] Error 2
```

so it seems `--platform=local` does not really work on m1 macs 😅

Reply:
thinking out loud, why do we even need this `--platform=linux/x86_64` in this particular target ? Other docker building commands do not specify this